tests/framework/cli/pipeline/test_pipeline_package.py

```diff
@@ -334,6 +334,52 @@ def test_package_modular_pipeline_with_nested_parameters(
         assert "retail/config/parameters/retail.yml" in wheel_contents
         assert "retail/config/parameters/retail_banking.yml" not in wheel_contents
 
+    def test_package_pipeline_with_deep_nested_parameters(
```

Comment:
Maybe worth updating this test (or making a new one) to cover a more deeply nested version, just to check our glob is working as expected. Like `"conf" / "base" / "parameters" / "a" / "b" / "c"/ "d"`

Reply:
Done!